src/LRSResponse.php

```diff
@@ -30,4 +30,16 @@ public function __construct($success, $content, $httpResponse) {
         $this->content = $content;
         $this->httpResponse = $httpResponse;
     }
+
+    public static function fromRemoteLRSResponse(array $response, array $options, $content) {
```

Comment:
Is there a reason to move this function here (and to make it a function at all)? Does it serve a purpose in the PHP7 changes?